lib/ip_record.js

```diff
@@ -123,6 +124,32 @@ module.exports = function (limits, now) {
     this.as = this._trim(now, this.as, limits.maxAccountStatusCheck)
   }
 
+  IpRecord.prototype.isOverSMSLimit = function () {
+    this.trimSMSRequests(now())
```

Comment:
...oh you've capitalised them here as well. I don't know, feel free to ignore what I said above in that case, just as long as it's upper-cased everywhere. A sometimes-uppercase-but-sometimes-lowercase mish-mash would keep me awake at night.